packages/devops/Dockerfile

```diff
@@ -0,0 +1,104 @@
+FROM node:14.19.3-alpine3.15 as builder
```

Comment:
Node version matches engine in package.json, image matches whats used in codeship